README.md

```diff
@@ -1,53 +1,42 @@
-## GPII Flow Manager Dockerfile
+# GPII Flow Manager Dockerfile - Ansible version
 
+Builds a GPII Preferences Server Docker container image. This image is built using the [Ansible role](https://github.com/gpii-ops/ansible-flow-manager).
 
-This repository is used to build [GPII Flow Manager](http://wiki.gpii.net/w/Architecture_Overview#Flow_Manager) Docker images.
+## Building
 
+- build Ansible-provisioned image:
+  - `docker build --no-cache -t gpii/flow-manager .`
 
-### Environment Variables
+## Runtime Environment Variables
 
-The following environment variables can be used to affect the container's behaviour:
+- `PREFERENCES_SERVER_HOST_ADDRESS`: host address of the preferences server instance to use. (default: `preferences.gpii.net`)
+- `NODE_ENV`: specifies the configuration file to be used from https://github.com/GPII/universal/tree/master/gpii/configs when launching (default: `cloudBased.production`)
+- `CONTAINER_TEST`: whether or not to run the container in test mode, then exit (default: `false`)
 
-* `NODE_ENV` - setting this variable is required and there is no default, one possible option is `cloudBased.production`
+## Testing
 
-* `PREFERENCES_SERVER_HOST_ADDRESS` - this is optional and defaults to `preferences.gpii.net` 
+The container can be tested as part of a GPII deployment by setting the *CONTAINER_TEST* environment variable to *true*.
 
-* `RBMM_HOST_ADDRESS` - this is optional and defaults to `rbmm.gpii.net` 
+This mode is typically expected to connect to a running preferences server with the 'Carla' test data set. The container will exit after the test and the exit code as a reuslt of the run command can be used for further actions. The container can (to a certain extent) self-test using the development mode - see the run examples below - but this obviously doesn't test the real production runtime configuration.
 
-* `STMM_HOST_ADDRESS` - this is optional and defaults to `stmm.gpii.net` 
+## Running
 
+- running requires a preferences server instance accessible to the container. As in the example below, this can be a self-contained preferences server running in the same container in development mode.
```

Comment:
Suggestion:

" A Preferences Server instance will need to be accessible to the Flow Manager container. As in the example below, this can be a self-contained Preferences Server running in the same container in development mode."